nursery/coin_select/src/coin_selector.rs

```diff
@@ -35,7 +35,8 @@ impl Default for Target {
     fn default() -> Self {
         Self {
             feerate: FeeRate::default_min_relay_fee(),
-            min_fee: 0, // TODO figure out what the actual network rule is for this
+            // https://bitcoin.stackexchange.com/questions/69282/what-is-the-min-relay-min-fee-code-26
+            min_fee: 1000,
```

Comment:
I think the `relayfee` mentioned in that link is rather a feerate given in BTC/kvB:
https://github.com/bitcoin/bitcoin/blob/82ea4e787c791acbc85fd3043dd6bae038cba4f2/src/rpc/net.cpp#L643

Reply:
Thanks for pointing this out. Definitely a bad mistake.

Reply:
Outside of replacements there is no minimum absolute fee rule. It's implicit from the minimum feerate (1 sat/vb) and how transactions cannot have a size of 0.